docs/source/srs.rst

```diff
@@ -0,0 +1,283 @@
+involved. Below is a breakdown of the intended audiences and how they may use
+the document. This list is not exhaustive, but intended to guide common readers
+of this document.
+
+* Electrical Team Members: Members of the electrical team who are designing
+  and building the DEV1 IMU. Members of this group may refer to this document
+  to ensure requirements and constraints align with their expectations.
+  Additionally, this document can be used as a point of reference during
+  the hardware/firmware bring up and debugging.
+* Firmware Team Members: Developers on the firmware team who are designing,
+  developing, and testing the DEV1 IMU firmware. Members of this team will
+  need to refer to this document throughout the development process to ensure
+  all target needs are met within the agreed upon constraints.
+* Integration Team Members: RIT EVT team members who handle systems level
+  integration on the team. These team members may use this document to gain
+  and understanding of how the DEV1 IMU will operate. Most critically, how the
+  DEV1 IMU will operate within the structure of the DEV1 architecture.
+
+The document is laid out where it is not strictly necessary to read all sections
+in their entirety. Instead each section should be self-contained and 
+comprehensive within its scope. As such audience members are encouraged to read 
+the sections that most pertain to their needs. Additionally, the glossary in the
+Appendix may prove useful in situations where phrases may be used in specific 
+ways within this document.
+
+Product Scope
+-------------
+
+The DEV1 IMU measures acceleration and position data for the DEV1 bike. The
+DEV IMU reports its measurements over the DEV1 CAN network to provide a
+general overview of the bike's orientation for other boards to act
+accordingly for optimal performance.
+
+Overview
+--------
+
+The rest of this software design document will go further into the specifics of 
+the requirements while also looking at the constraints of the system. The goal 
+is to clarify the use cases of the DEV1 IMU and to specify what the DEV1 IMU 
+will do in those cases. Design considerations will not be discussed, however 
+notable design constraints will be covered.
+
+
+Overall Description
+===================
+
+Product Perspective
+-------------------
+
+The DEV1 IMU plays a role in performance optimization for the DEV1 bike. It
+reports measurements such as linear/angular acceleration and Euler over the
+bike's CAN network, allowing other boards to adjust their configurations to
+maintain the best performance.
+
+The DEV1 IMU is made up of two components, the STM32F334K8 microcontroller
+and the BNO055 orientation sensor chip. The ST microcontroller handles
+processing of data and manages CAN network communication for the IMU. The
+BNO055 is used to obtain the orientation measurements.
+
+* list some boards that will use the data
```

Comment:
fix pls.

Reply:
For DEV1, this data is not currently being used by other boards, but it can be used for traction control in the future